gradle/fork/gradle.properties.peb

```diff
@@ -16,9 +16,9 @@ instance.local-publish.jvmOpts={{aemInstanceJvmOpts}}
 {% endif %}
 
 {% if aemInstanceType == 'local' %}
-localInstance.zipUrl={{aemLocalInstanceZipUri}}
-localInstance.jarUrl={{aemLocalInstanceJarUri}}
-localInstance.licenseUrl={{aemLocalInstanceLicenseUri}}
+localInstance.backup.downloadUrl={{aemLocalInstanceZipUri}}
```

Comment:
So backup it will have prefix localInstance?

Reply:
yep, you cannot backup remotes ;)

and it is somehow paired with 

`localInstance.quickstart` and `localInstance.backup` ; looks nice in my opinion;

one thing for creating from scratch, second for restoring